hasher-matcher-actioner/hmalib/lambdas/api/submit.py

```diff
@@ -233,9 +244,15 @@ def get_submit_api(
     submit_api = bottle.Bottle()
     s3_bucket_image_source = S3BucketContentSource(image_bucket, image_prefix)
 
+    def _content_exist_error(content_id: str):
+        return bottle.abort(
+            500,
```

Comment:
Not that we are super sensitive about status codes, but this is a 400 Bad Request or a 409 Conflict. Upto you, but a 4xx is more appropriate here.